VirtoCommerce.CatalogModule.Web/Scripts/blades/property-list.js

```diff
@@ -188,15 +191,36 @@ angular.module('virtoCommerce.catalogModule')
                 name: "catalog.blades.property-list.labels.reset-filter", icon: 'fa fa-undo',
                 executeMethod: function () {
                     if ($localStorage.entryPropertyFilters) {
-                        $localStorage.entryPropertyFilters[authService.id] = [];
+                        
```

Comment:
Probably removing `$localStorage.entryPropertyFilters[authService.id] = [];` could lead to errors if there were no filters saved.